crates/socket/Cargo.toml

```diff
@@ -26,9 +26,10 @@ async-net = "1.4.3"
 tokio = { version = "0.2.21", features = ["macros"] }
 tokio-util = { version = "0.3.1", features = ["codec", "compat"] }
 async-trait = "0.1.21"
+thiserror = "1.0.20"
 
 # Fluvio dependencies
-fluvio-future = { version = "0.1.0", features = ["tls", "net", "zero_copy"] }
```

Comment:
don't need to change this, cargo will automatically dependencies

Reply:
Well I need to make sure I have at least `0.1.2` or this won't build. In the `fluvio-future` 0.1.2 build I made the error types implement `std::error::Error` which is required to include them in `thiserror`.

Reply:
ok